packages/yourloops/app/main-lobby.tsx

```diff
@@ -65,46 +66,51 @@ const routeStyle = makeStyles<Theme>(() => {
   }
 })
 
+export const isRoutePublic = (route: string): boolean => {
+  return PUBLIC_ROUTES.includes(route)
+}
```

Comment:
Why export this function ? And what about a one liner ?
```suggestion
export const isRoutePublic = (route: string): boolean => PUBLIC_ROUTES.includes(route)
```

Reply:
Function is exported for tests. I don't like one liner